modules/ROOT/pages/_partials/csv-inbound-partial.adoc

```diff
@@ -0,0 +1,20 @@
+// CSV settings for the message types
+
+Use the following settings to define CSV as the format for the {message-type} message type:
+
+[%header%autowidth.spread]
+|===
+| Setting | Description | Default | Required
+| Usage | Message usage. Allowed value is `{message-type}`. | | Yes
+| Format | Format of the message. Allowed value is `CSV`.| | Yes
+| Separator character | Character that separates values in the file. | , |Yes
+| Quote character | Character used as a quote character in the file.  | "" |Yes 
+| Escape character | Character used as an escape character in the file. | \ | Yes 
+| Header line | When selected, Partner Manager skips the header row when it processes a CSV file. | Selected |
```

Comment:
Do you need "When selected"? Can you say "Skip header row when processing CSV files."?

Reply:
Leaving as is